README.md

```diff
@@ -17,41 +17,53 @@
 </p>
 
 
-Transformers Interpret is a model explainability tool designed to work exclusively with 🤗 transformers.
+Transformers Interpret is a model explainability tool designed to work exclusively with the [transformers][transformers] package.
```

Comment:
That emoji looks weird if someone doesn't realise that the `transformers` package uses it too.

Reply:
It's a strange one. Because Transformers is actually a general architecture in ML/NLP the  🤗  has become a strong identifier in conversations that you are talking about the package and not the architecture. I do like adding the link here though but think I'll leave the emoji.

Reply:
Fair enough!